src/components/Title/index.js

```diff
@@ -1,7 +1,7 @@
+import { StyledTitle } from "./title.styles";
+
 const Title = (props) => {
-  return (
```

Comment:
I'm sure you know how to do this a little bit better by now 😄

Reply:
😂